shambomon/lib/shambomon/game.ex

```diff
@@ -172,42 +184,78 @@ defmodule Shambomon.Game do
     Map.put(game, :messages, msgs)
   end
 
+  defp update_messages(game, player, special_roll, _special?) do
```

Comment:
The special attack message is currently only printing for P2, but not P1.

Reply:
Actually, it's printing for both now. I had it only print for one player the first 3 times I tested it. Lemme see if we're missing a case.

Reply:
Okay, it doesn't print P1's special attack message if P1 uses it in the first round.

Reply:
There might be a case when one player fails to roll a 6 for their special and the round ends in a tie.

Reply:
Ah okay, you're right. We should still print the special attack message, though, so that the players know it was used.

Reply:
Oh oops I didn't see your previous message...I meant that I thought there might be a case at any point during the game where one player fails a special attack and both players chose the same default attack. But idk if that's the exact case

Reply:
And hmm so it doesn't print any special message if it's used in the first round?

Reply:
No, it doesn't print in the case that you mentioned. I just happened to choose the same default attack in the first round, so I thought it was because it was the first round.

Reply:
Just pushed a fix for this.